namespaces/live-1.cloud-platform.service.justice.gov.uk/authorized-keys-provider/provider.yaml

```diff
@@ -0,0 +1,115 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: python-key-provider
+  namespace: authorized-keys-provider
+data:
+  authorized-keys-generator: |
+    #!/usr/bin/env python3
+
+    import os
+    import sys
+    import boto3
+    from github import Github
+
+    def getenv(var):
+        val = os.environ.get(var)
+        if val is None or val == '':
+            print('{} is not set, cannot continue'.format(var))
+            sys.exit(1)
+        return val
+
+    def main():
+        token = getenv('GITHUB_TOKEN')
+        org_name = getenv('GITHUB_ORG')
+        team_name = getenv('GITHUB_TEAM')
+        bucket = getenv('S3_BUCKET_NAME')
+
+        authorized_keys = ''
+        org = Github(token).get_organization(org_name)
+        for team in org.get_teams():
+            if team.name.lower() == team_name:
+                for member in team.get_members():
+                    authorized_keys += "# {}\n".format(member.name)
+                    for k in member.get_keys():
+                        authorized_keys += "{} @{}\n".format(k.key, member.login)
+                break
+        boto3.resource('s3').Bucket(bucket).Object('authorized_keys').put(
+            ACL='public-read', Body=authorized_keys, ContentType='text/plain')
+
+    if __name__ == '__main__':
+        main()
+
+
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  labels:
+    app: authorized-keys-provider
+  name: authorized-keys-provider
+  namespace: authorized-keys-provider
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: authorized-keys-provider
+  template:
+    metadata:
+      labels:
+        app: authorized-keys-provider
+    spec:
+      containers:
+      - name: alpine
```

Comment:
Does the alpine image run as root by default, if so, it won't pass the psp requirements.

Reply:
@jasonBirchall Thanks for identifying this, updated this to use ministryofjustice/cloud-platform-tools:1.4 and added securityContext, to run as non root user.